src/Raven.Client/ServerWide/Operations/ReorderDatabaseMembersOperation.cs

```diff
@@ -15,20 +15,22 @@ public class ReorderDatabaseMembersOperation : IServerOperation
         public class Parameters
         {
             public List<string> MembersOrder;
+            public bool Fixed;
         }
 
         private readonly string _database;
         private readonly Parameters _parameters;
 
-        public ReorderDatabaseMembersOperation(string database, List<string> order)
+        public ReorderDatabaseMembersOperation(string database, List<string> order, bool fixedTopology = false)
```

Comment:
This breaks binary compatibility. Please add a new constructor for that overload.

Reply:
done